install.sh

```diff
@@ -77,7 +77,7 @@ sudo yum install jq-1.5*
 echo "================= Installing Node 8.x ==================="
 . /c7/node/install.sh
 
-echo "================= Installing Java 10.x ==================="
+echo "================= Installing Java 11 ==================="
```

Comment:
Please remove the version numbers from here. The actual version number is coming from the install.sh file, so you don't really know here what it's going to install.

Reply:
ok